fs_xgspon_mod.py

```diff
@@ -757,7 +781,7 @@ def parse_vlan_filter(vf):
     parse_install.add_argument("--eqvid", type=parse_length(20))
     parse_install.add_argument("--hwver", type=parse_length(14))
     parse_install.add_argument("--swver", type=parse_length(14))
-    parse_install.add_argument("--eth_slot", type=int, choices=(1, 10))
+    parse_install.add_argument("--eth_slot", type=int, choices=(range(1, 65534)), metavar="[1-65534]")
```

Comment:
Unfortunately, the slots need to be particularly constrained. Some of them have hardcoded purposes (can be found in both the `MecMgr` and `EthMgr` binaries) and while it's been a while if I remember it is used directly as an array index in some places where too-high values would cause a crash. 1 and 10 are known safe values, and I'd need to confirm manually what other values are safe (but I believe all safe values are between 1 and 10).